front-end/pages/legal.vue

```diff
@@ -0,0 +1,7 @@
+<template>
```

Comment:
Instead of `legal`, add three pages, `Privacy Policy`, `Distribution Rights` and `Terms and Conditions`.